specification/dimensions/billingaccountid.md

```diff
@@ -0,0 +1,32 @@
+# Billing Account ID
+
+A billing account is a container for resources and/or services that are billed together in an invoice. Common scenarios for using billing accounts include grouping based on organizational constructs and hierarchy-based cost allocation approach.
+
+A billing account ID is an provider assigned identifier for a billing account. It is commonly used for cost reporting, invoice reconciliation and cost allocation scenarios.
+
+The BillingAccountId column MUST be present in the billing data. This column must be of type String and MUST NOT contain null values. BillingAccountId SHOULD be a globally unique identifier within a provider. BillingAccountId MUST be unique when a customer has more than one billing account.
```

Comment:
```suggestion
The BillingAccountId column MUST be present in the billing data. This column must be of type String and MUST NOT contain null values. BillingAccountId MUST be a globally unique identifier within a provider.
```
Had some discussions. I will change my vote and go with some of the others that suggested simplifying this to say MUST be globally unique.